From 5dbeb8a9344b4fe572c184fb1d0bede1ae85c557 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <mathieu@hashbang.fr>
Date: Wed, 8 Jan 2020 14:26:15 +0100
Subject: [PATCH] Improvements to appdata

- Add OARS info
- Follow the new recommendation for app ids
- Add a reference to the old app id
- Add release descriptions
---
 .../appdata/ch.x29a.playitslowly.appdata.xml  | 33 ++++++++++++++++++-
 1 file changed, 32 insertions(+), 1 deletion(-)

diff --git a/share/appdata/ch.x29a.playitslowly.appdata.xml b/share/appdata/ch.x29a.playitslowly.appdata.xml
index 0b754f0..28ec42b 100644
--- a/share/appdata/ch.x29a.playitslowly.appdata.xml
+++ b/share/appdata/ch.x29a.playitslowly.appdata.xml
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <application>
-  <id type="desktop">ch.x29a.playitslowly.desktop</id>
+  <id type="desktop">ch.x29a.playitslowly</id>
+  <launchable type="desktop-id">ch.x29a.playitslowly.desktop</launchable>
+  <provides>
+    <id>playitslowly.desktop</id>
+  </provides>
   <name>Play it Slowly</name>
   <summary>Play music at a different speed</summary>
   <description>
@@ -9,9 +13,36 @@
     </p>
   </description>
   <url type="homepage">https://29a.ch/playitslowly/</url>
+  <content_rating type="oars-1.1" />
   <screenshots>
     <screenshot type="default">https://29a.ch/playitslowly/screenshot.png</screenshot>
   </screenshots>
   <project_license>GPL-3.0+</project_license>
   <metadata_license>CC0-1.0</metadata_license>
+  <releases>
+    <release version="1.5.1" date="2016-05-21">
+      <description>
+        <p>This is a bug fix release:</p>
+        <ul>
+          <li>Fixed config loading</li>
+        </ul>
+      </description>
+    </release>
+    <release version="1.5.0" date="2016-05-21">
+      <description>
+        <p>This is a major release:</p>
+        <ul>
+          <li>Updated to Python3, GTK3 and GStreamer 1.0</li>
+          <li>Minor design cleanup</li>
+          <li>Fixed a bug where the file chooser would not show the currently selected file</li>
+        </ul>
+      </description>
+    </release>
+    <release version="1.4.0" date="2011-07-03" />
+    <release version="1.3.1" date="2010-12-08" />
+    <release version="1.3.0" date="2010-07-04" />
+    <release version="1.2.0" date="2009-05-23" />
+    <release version="1.1.0" date="2009-01-08" />
+    <release version="1.0.0" date="2008-12-11" />
+  </releases>
 </application>
-- 
2.37.1

